tests/app/dao/notification_dao/test_notification_dao_performance_platform.py

```diff
@@ -0,0 +1,94 @@
+from datetime import date, datetime, timedelta
+
+from freezegun import freeze_time
+
+from app.dao.notifications_dao import dao_get_total_notifications_sent_per_day_for_perfomance_platform
+from app.models import KEY_TYPE_NORMAL, KEY_TYPE_TEAM, KEY_TYPE_TEST
+
+from tests.app.db import create_notification
+
+
+def test_get_total_notifications_filters_on_date(sample_template):
+    create_notification(sample_template, created_at=datetime(2016, 10, 17, 10, 0))
+    create_notification(sample_template, created_at=datetime(2016, 10, 18, 10, 0))
+    create_notification(sample_template, created_at=datetime(2016, 10, 19, 10, 0))
+    result = dao_get_total_notifications_sent_per_day_for_perfomance_platform(date(2016, 10, 18), date(2016, 10, 19))
+    assert result.messages_total == 1
+
+
+def test_get_total_notifications_filters_on_date_at_midnight(sample_template):
+    create_notification(sample_template, created_at=datetime(2016, 10, 17, 23, 59, 59))
+    create_notification(sample_template, created_at=datetime(2016, 10, 18, 0, 0))
+    create_notification(sample_template, created_at=datetime(2016, 10, 18, 23, 59, 59))
+    create_notification(sample_template, created_at=datetime(2016, 10, 19, 0, 0))
+    result = dao_get_total_notifications_sent_per_day_for_perfomance_platform(date(2016, 10, 18), date(2016, 10, 19))
+    assert result.messages_total == 2
+
```

Comment:
Can we combine`test_get_total_notifications_filters_on_date` and `test_get_total_notifications_filters_on_date_at_midnight`, seems like they're just testing that returned notifications are within the date range.